code/modules/religion/religion_structures.dm

```diff
@@ -0,0 +1,121 @@
+/obj/structure/altar_of_gods
+	name = "\improper Altar of the Gods"
+	desc = "An altar which allows the head of the church to choose a sect of religious teachings as well as provide sacrifices to earn favor."
+	icon = 'icons/obj/hand_of_god_structures.dmi'
+	icon_state = "convertaltar"
+	density = TRUE
+	anchored = TRUE
+	layer = TABLE_LAYER
+	climbable = TRUE
+	pass_flags = LETPASSTHROW
+	can_buckle = TRUE
+	buckle_lying = 90 //we turn to you!
+	var/datum/religion_sect/sect_to_altar // easy access!
+	var/datum/religion_rites/performing_rite
+
+/obj/structure/altar_of_gods/examine(mob/user)
+	. = ..()
+	var/can_i_see = FALSE
+	if(isobserver(user))
+		can_i_see = TRUE
+	else if(isliving(user))
+		var/mob/living/L = user
+		if(L.mind?.holy_role)
+			can_i_see = TRUE
+
+	if(!can_i_see || !sect_to_altar)
+		return
+
+	. += "<span class='notice'>The sect currently has [round(sect_to_altar.favor)] favor with [GLOB.deity].</span>"
+	if(!sect_to_altar.rites_list)
+		return
+	. += "List of available Rites:"
+	. += sect_to_altar.rites_list
+
+
+/obj/structure/altar_of_gods/Initialize(mapload)
+	. = ..()
+	if(GLOB.religious_sect)
+		sect_to_altar = GLOB.religious_sect
+		if(sect_to_altar.altar_icon)
+			icon = sect_to_altar.altar_icon
+		if(sect_to_altar.altar_icon_state)
+			icon_state = sect_to_altar.altar_icon_state
+
+/obj/structure/altar_of_gods/attack_hand(mob/living/user)
+	if(!Adjacent(user) || !user.pulling)
+		return ..()
+	if(!isliving(user.pulling))
+		return ..()
+	var/mob/living/pushed_mob = user.pulling
+	if(pushed_mob.buckled)
+		to_chat(user, "<span class='warning'>[pushed_mob] is buckled to [pushed_mob.buckled]!</span>")
+		return ..()
+	to_chat(user,"<span class='notice>You try to coax [pushed_mob] onto [src]...</span>")
+	if(!do_after(user,(5 SECONDS),target = pushed_mob))
+		return ..()
+	pushed_mob.forceMove(loc)
+	return ..()
+
+/obj/structure/altar_of_gods/attackby(obj/item/C, mob/user, params)
+	//If we can sac, we do nothing but the sacrifice instead of typical attackby behavior (IE damage the structure)
+	if(sect_to_altar?.can_sacrifice(C,user))
```

Comment:
```suggestion
	if(sect_to_altar?.can_sacrifice(C,user)) //If we can sac, we do nothing but the sacrifice instead of typical attackby behavior (IE damage the structure)
```